.env.example

```
@@ -72,6 +72,8 @@ OPENHUMAN_MODEL=
 OPENHUMAN_WORKSPACE=
 # [optional] Default: 0.7
 OPENHUMAN_TEMPERATURE=0.7
+# [optional] Local safety cap for side-effecting tool actions in a rolling hour (default 20)
+OPENHUMAN_MAX_ACTIONS_PER_HOUR=20
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The OPENHUMAN_MAX_ACTIONS_PER_HOUR key should go before the OPENHUMAN_MODEL key

(UnorderedKey)